PURL-TYPES.rst

```diff
@@ -578,6 +578,25 @@ swift
       pkg:swift/github.com/Alamofire/Alamofire@5.4.3
       pkg:swift/github.com/RxSwiftCommunity/RxFlow@2.12.4
 
+
+nix
+---
+
+``nix`` for Nix derivations:
+
+- The default package repository is <https://github.com/NixOS/nixpkgs> but the ``repository`` qualifier can override the package repository.
+- The ``name`` is the package ``pname`` field of the given derivation. It is case sensitive.
+- The ``version`` is the ``version`` field of the given derivation.
```

Comment:
How do PURL look like for store paths where the version cannot be extracted? Is this field (and other fields) optional or required? Other examples do explicitly say what's required and what not.